src/app/src/components/FacilityDetailSidebar.jsx

```diff
@@ -95,7 +96,10 @@ const detailsSidebarStyles = Object.freeze({
 
 class FacilityDetailSidebar extends Component {
     componentDidMount() {
-        return this.props.fetchFacility();
+        return this.props.fetchFacility(
+            Number(this.props.embed),
+            this.props.user.user.id,
```

Comment:
I was hoping to bind these in `mapDispatchToProps`, but as far as I could tell, you can't reference props that are bound in `mapStateToProps` in `mapDispatchToProps`.